README.md

```diff
@@ -21,7 +21,7 @@ You should have all of the following installed before installing GridKit™
     - [SUNDIALS](https://github.com/LLNL/sundials) >= 7.4.0 (optional)
         - To support sparse linear algebra, SUNDIALS must also be built with [KLU support](https://sundials.readthedocs.io/en/latest/sundials/Install_link.html#cmakeoption-ENABLE_KLU). You most likely want this.
     - [Ipopt](https://github.com/coin-or/Ipopt) >= 3.x (optional)
-    - [Enzyme](https://github.com/EnzymeAD/Enzyme) >=0.0.196 (optional). Note
+    - [Enzyme](https://github.com/EnzymeAD/Enzyme) == main at e954161 (optional). Note
       that the use of Enzyme is experimental, and some versions of it have been found to break GridKit code.
```

Comment:
```suggestion
    - [Enzyme](https://github.com/EnzymeAD/Enzyme) == main at e954161 or later (optional). Note
that the use of Enzyme is experimental, and some versions of it have been found to break GridKit code.
```

Reply:
we just made a release so we could alternatively recommend the release here [and potentially update spack with it]